### PR TITLE
made jlist clickable and when click takes to place on googlemaps

### PR DESCRIPTION
**Pull request by YaffaLevy**

### CODE REVIEW

**Comment by andrewoid** [comment]
Go back and fix all the checkstyle errors you had in your previous two PRs

**Comment by YaffaLevy** [comment]
how do i fidn my old checkstyles

**Comment by andrewoid** [comment]
> how do i fidn my old checkstyles

It's in the PRs you merged before I approved them.

**Comment by YaffaLevy** [comment]
i did not mean to do that. I fixed them

File: src/main/java/levy/earthquake/json/EarthquakeFrame.java [hunk 1]
```diff
@@ -10,12 +9,18 @@
...
 import java.awt.*;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.Arrays;
 
 public class EarthquakeFrame extends JFrame {
     private JList<String> jlist = new JList<>();
     private Disposable disposable;
+    public Feature[] features;
```
Comment (andrewoid) [inline src/main/java/levy/earthquake/json/EarthquakeFrame.java]: private

**Review by andrewoid — APPROVED** [review]

**Pull request merged**